components/eamxx/src/physics/shoc/tests/infra/shoc_data.hpp

```diff
@@ -67,17 +67,13 @@ struct FortranDataIterator {
   void init(const FortranData::Ptr& d);
 };
 
-// Initialize SHOC with the given number of levels.
-void shoc_init(Int nlev, bool use_fortran=false, bool force_reinit=false);
-
 // We will likely want to remove these checks in the future, as we're not tied
 // to the exact implementation or arithmetic in SHOC. For now, these checks are
 // here to establish that the initial regression-testing code gives results that
 // match the python f2py tester, without needing a data file.
 Int check_against_python(const FortranData& d);
```

Comment:
It appears that this function (like the one with the same name in p3) is never defined. Let's prune this decl too.